src/main/java/com/cloudogu/wiki/scmm/ScmManager.java

```diff
@@ -51,7 +52,7 @@ private ScmManager() {
 
             return wikis;
         } catch (UnirestException ex) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS1166)